app.rb

```diff
@@ -0,0 +1,70 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require './lib/peep'
+require './lib/user'
+require_relative 'chitter_helpers'
+require_relative 'database_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  register Sinatra::Flash
+  helpers ChitterHelpers
+
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps' do
```

Comment:
You could separate all your peeps routes into a separate Peeps controller and your sessions into a sessions controller etc.